docs/sphinx/source/whatsnew/v0.11.0.rst

```diff
@@ -15,6 +15,10 @@ Deprecations
 
 Enhancements
 ~~~~~~~~~~~~
+* Added conversion functions from spectral response :math:`[A/W]` to quantum
+  efficiency :math:`[unitless]` and vice versa. The conversion functions are
+  :py:func:`spectral_responsivity_to_quantum_efficiency` and :py:func:`quantum_efficiency_to_spectral_responsivity` respectively.
+  (:issue:`2040`, :pull:`2041`)
```

Comment:
I guess these will need the full names (e.g. `pvlib.spectrum.spectral_responsivity_to_quantum_efficiency`)